src/screens/WalletScreen/index.js

```diff
@@ -101,6 +105,10 @@ class WalletScreen extends NavigatorComponent<Props & TestingModeProps & Actions
     this.props.navigator.push(screen('RECEIVE_MONEY_SCREEN'));
   };
 
+  onRefresh = () => {
+    this.props.isRefreshing = true;
```

Comment:
I think changing props of component is not an expected way to do that. Use state for that. 
Be sure to set it to false when refresh is done.